packages/hub/routes/card-spaces.ts

```diff
@@ -148,9 +148,19 @@ export default class CardSpacesRoute {
       return;
     }
 
-    cardSpace.profileDescription = this.sanitizeText(ctx.request.body.data.attributes['profile-description']);
-    cardSpace.profileImageUrl = this.sanitizeText(ctx.request.body.data.attributes['profile-image-url']);
-    cardSpace.links = ctx.request.body.data.attributes['links'];
+    let attributes = ctx.request.body.data.attributes;
+
+    if (attributes['profile-description']) {
+      cardSpace.profileDescription = this.sanitizeText(ctx.request.body.data.attributes['profile-description']);
+    }
+
+    if (attributes['profile-image-url']) {
+      cardSpace.profileImageUrl = this.sanitizeText(ctx.request.body.data.attributes['profile-image-url']);
+    }
+
+    if (attributes['links']) {
+      cardSpace.links = ctx.request.body.data.attributes['links'];
```

Comment:
Links is effectively a PUT for now, but I don't know if that is a problem we need to handle right now.

Reply:
ya… I think for now sadly we just need the client to send everything in `links` 🥴